package.json

```diff
@@ -39,7 +39,8 @@
     "strip-json-comments": "^5.0.1"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=16",
+    "npm": ">=8"
```

Comment:
The tests only [cover 16 & 18](https://github.com/h5bp/html5-boilerplate/blob/87bdc80ab9e35dcb52d75ffc926d8ed67991c626/.github/workflows/test.yaml#L17) so I assumed that should be the minimums for the engines specification.